scripts/create-db-cleanup-service.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cf target -s preview
+DB_PLAN=$(cf service digitalmarketplace_api_db | grep -i 'plan: ' | cut -d ' ' -f2)
+cf target -s db-cleanup
+[ $(cf target | grep -i 'space' | cut -d':' -f2) = "db-cleanup" ] || (echo "Error: This can only be run in the db-cleanup space" && exit 1)
+cf create-service postgres ${DB_PLAN} digitalmarketplace_db_cleanup
```

Comment:
Yeah, I think the `sleep 10`s thing was me misunderstanding cloud foundry and wasn't actually needed. When switching spaces it turns out it does happen in a subshell which only returns when it's done, so it's all good.